uniffi_bindgen/src/bindings/swift/templates/EnumTemplate.swift

```diff
@@ -1,30 +1,40 @@
-public enum {{ e.name()|class_name_swift }}: ViaFfi {
+public enum {{ e.name()|class_name_swift }}: ViaFfiUsingByteBuffer, ViaFfi, Equatable {
```

Comment:
Sounds good, I filed https://github.com/mozilla/uniffi-rs/issues/396 for future discussions.

Reply:
That makes sense. I think in practice they will probably get an error before they even try to generate swift code, because the Rust code will need to use some `Box` indirection and the generated scaffolding won't support it. But I can add a link to the issue in a few key places in the hope that any such users will find it.